### DISCUSSION TITLE
Typescript with ESBuild config error

### DISCUSSION THREAD

**Discussion by charlotteclough** [badeball/cypress-cucumber-preprocessor#945] [Q&A]
Hi,

I'm trying to setup the pre-processor using Typescript and the ESBuild bundler as outlined [here](https://github.com/badeball/cypress-cucumber-preprocessor/blob/master/examples/esbuild-ts/cypress.config.ts)

However when I follow this I get the following config error:
![image](https://user-images.githubusercontent.com/78296140/218090082-aa391259-c930-402a-bea5-c4d14545b8c7.png)
![image](https://user-images.githubusercontent.com/78296140/218090612-37c6ad2e-7dac-4d70-8cb3-cf8e55b6ebef.png)
any ideas on how to solve?

**Comment by badeball** [comment]
This is answered in the FAQ.
